src/contract-dumps.ts

```diff
@@ -153,6 +153,9 @@ export const makeStateDump = async (): Promise<any> => {
     OVM_ECDSAContractAccount: '0x4200000000000000000000000000000000000003',
     OVM_ProxySequencerEntrypoint: '0x4200000000000000000000000000000000000004',
     OVM_SequencerEntrypoint: '0x4200000000000000000000000000000000000005',
+    //L2 ETH at 0x4200000000000000000000000000000000000006
```

Comment:
Is `L2_ETH` not deployed as part of this?

Reply:
No, not until we get the L2 dump stuff fixed here https://github.com/ethereum-optimism/roadmap/issues/157